pygotham/frontend/__init__.py

```diff
@@ -23,6 +24,7 @@ def create_app(settings_override=None):
 
     assets = Environment(app)
     Foundation(app)
+    Sentry(app)
```

Comment:
Yep. From my logs:

    web_1 | Raven is not configured (logging is disabled). Please see the documentation for more information.